components/SearchForm.vue

```diff
@@ -1,12 +1,27 @@
 <template>
 	<div class="search-container">
 		<form id="search-form" action="/search">
-			<input
-				name="s"
-				type="search"
-				placeholder="Search..."
-				aria-label="Enter keywords for a site-wide search"
-			>
+			<input :placeholder="placeholder" :aria-label="ariaLabel" name="s" type="search">
+			<input :value="context" type="hidden" name="context">
```

Comment:
Is this correct how there are two inputs elements here?

Reply:
Having two input elements is for both of them to be included in the URL at the form submission so that the search page knows the scope that the search should be performed. Now the search script handles 2 cases:

1. if the `context` value in the query is empty, search across the site;
2. if the `context === "views"`, only search thru views posts.